front-end-side/src/Pages/Data.js

```diff
@@ -142,6 +142,10 @@ const EnterData = () => {
 
     useEffect(() => {
         if (categoryStatus) {
+
+            setPreviousMonthExpenses(categoryData.data.previousCarSum + categoryData.data.previousClothesSum + categoryData.data.previousEntertaintmentSum + categoryData.data.previousFoodSum + categoryData.data.previousOtherSum + categoryData.data.previousHouseSum);
+            setPreviousYearExpenses(categoryData.data.previousYearCarSum + categoryData.data.previousYearClothesSum + categoryData.data.previousYearEntertaintmentSum + categoryData.data.previousYearFoodSum + categoryData.data.previousYearOtherSum + categoryData.data.previousYearHouseSum);
```

Comment:
We have ```setPreviousMonthExpenses``` and ```setPreviousYearExpenses``` in ```sendDate``` function, are they needed there?

Reply:
Oh, I missed it, thank u